storage.bs

```diff
@@ -178,6 +178,7 @@ larger <a>site storage quota</a>. Factors such as navigation frequency, recency
 bookmarking, and <a href="#persistence">permission</a> for {{"persistent-storage"}} can be used as
 indications of "popularity".
 
+A <dfn export>site storage backend</dfn> is a specific site storage implementation.
```

Comment:
This doesn't really clarify anything. What is an implementation? Like, Chrome or Firefox or Safari? I think what you're getting at is what the spec refers to as "site".